src/utils.lua

```diff
@@ -1,7 +1,7 @@
-local base64 = require("base64")
-local crypto = require("crypto.init")
-local json = require("json")
-local constants = require("constants")
+local base64 = require(".src.base64")
+local crypto = require(".crypto.init")
```

Comment:
Should this be .src.crypto.init?

Reply:
when we first created the process, `crypto` was not available in the global scope as `.crypto.init`. it now is - I will have to check, but in that case we could remove our copy of crypto entirely

reference: https://github.com/ar-io/aos/tree/main/process/crypto

Reply:
related (and optional) - https://github.com/ar-io/ar-io-network-process/pull/396